### PR TITLE
JSON updates of Schm10 references. Closes #168

### LINKED ISSUE #168: Fix Schm10b Publication Record
Publication Schm10b is in the database twice, both as Schm10 and Schm10b.

### PR DESCRIPTION
**Pull request by kelle**

### CODE REVIEW

**Comment by Will-Cooper** [comment]
Have we documented this somewhere? Changing duplicate short refs to be NameYY.DOI? Or however it is we're doing this?

File: simple/schema.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Publications(Base):
     This stores reference information (DOI, bibcodes, etc) and has shortname as the primary key
     """
     __tablename__ = 'Publications'
-    name = Column(String(30), primary_key=True, nullable=False)
+    name = Column(String(30), primary_key=True, nullable=False, onupdate='cascade')
```
Comment (Will-Cooper) [inline simple/schema.py:16]: what is this doing?

**Review by Will-Cooper — COMMENTED** [review]
Python looks good

**Review by dr-rodriguez — APPROVED** [review]

File: simple/schema.py (hunk 1 above)
Comment (dr-rodriguez) [reply simple/schema.py:16]: I will un-do this after the PR since that shouldn't be there

**Pull request merged**